packages/system/victoria-metrics-operator/charts/victoria-metrics-operator/charts/victoria-metrics-common/README.md

```
@@ -1,3 +1,4 @@
 # VictoriaMetrics Common Helm chart
 
-Chart documentation is available [here](https://docs.victoriametrics.com/helm/victoria-metrics-common/)
+Chart documentation is available [here](https://docs.victoriametrics.com/helm/victoria-metrics-common/).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)